plugins/declarative/src/voicecallmanager.cpp

```diff
@@ -360,7 +369,7 @@ void VoiceCallManager::onPendingCallFinished(QDBusPendingCallWatcher *watcher)
     watcher->deleteLater();
 }
 
-void VoiceCallManager::onPendingSilenceFinished(QDBusPendingCallWatcher *watcher)
+void VoiceCallManager::onPendingVoidCallFinished(QDBusPendingCallWatcher *watcher)
```

Comment:
I may be misunderstanding the intention here, but is this a typo?
```suggestion
void VoiceCallManager::onPendingVoiceCallFinished(QDBusPendingCallWatcher *watcher)
```

Reply:
This was actually on purpose. Earlier there was onPendingCallFinished() and onPendingSilenceFinished() which were identical except the declaration of QDBusPendingReply which had type of the call. So instead of creating another finish handled I renamed the silence call one to more generic.

VoiceCallFinished could be mistaken as, heh, voice call being finished :)

Reply:
I see, so it's a reference to the return value, rather than the action taken. In this case, I think it would be good to update `onPendingCallFinished` to `onPendingBoolCallFinished` to make the intention clear.

Reply:
Sure, did that.